src/ess/powder/masking.py

```diff
@@ -71,3 +67,31 @@ def apply_masks(
 
 
 providers = (read_pixel_masks, apply_masks)
+
+
+def _merge(*dicts: dict) -> dict:
+    return {key: value for d in dicts for key, value in d.items()}
+
+
+def set_pixel_mask_filenames(
```

Comment:
```suggestion
def with_pixel_mask_filenames(
```
because it does not modify the input.

Reply:
@nvaytet Should we change all the naming of similar functions in ESSsans?

Reply:
fine by me

Reply:
Done